### DISCUSSION TITLE
Which .tflite file to use in latest rolling version?

### DISCUSSION THREAD

**Discussion by stalakerob** [jomjol/AI-on-the-edge-device#820] [Q&A]
Hi, there are several .tflite files in the actual rolling version:
dhy0030s3q.tflite
dhy0040s3q.tflite
effnet100-q.tflite
I don't understand when to use which. I'm using AIOTE to read the kwh of an analoge utility meter. For that I used the master branch so far but the issue is that the latest digit (after the comma) it mostly "N". I was hoping the new rolling version has improved but none of the new .ftlite  files work for me. Almost all digits are "N".

**Comment by jomjol** [comment]
for the analogue you should use the files beginning with "ana", for the digital beginning with "dig"
"dhy" and "effnet" are in testing for internal only.
